packages/dmn-js-decision-table/src/features/decision-table-head/editor/components/OutputEditor.js

```diff
@@ -33,12 +39,24 @@ export default class OutputEditor extends Component {
     }
   }
 
+  renderValidation() {
```

Comment:
I think it's not needed to add a custom validation component here. We already have the [`ValidatedInput`](https://github.com/bpmn-io/dmn-js/blob/develop/packages/dmn-js-shared/src/components/ValidatedInput.js) component available.

We already use it in other places: https://github.com/bpmn-io/dmn-js/search?q=ValidatedInput. E.g. for date inputs

![image](https://user-images.githubusercontent.com/9433996/95723099-0cc6e380-0c75-11eb-880f-0f65497203b0.png)